addons/a11y/src/components/Report/Info.tsx

```diff
@@ -4,6 +4,7 @@ import { styled } from '@storybook/theming';
 import { Result } from 'axe-core';
 
 const Wrapper = styled.div(({ theme }) => ({
+  // Hmm sadly background of SB theme has no `bar` property :/
```

Comment:
If my typing is right then `backgroundColor` is not properly defined, any idea what could be the correct value?

Reply:
Discussing with @ndelangen `bar` should be added and it's default value id  `#fff`